low-level/framework/utils/utility.py

```diff
@@ -58,3 +60,56 @@ def is_env_vm(self):
             logger.warning("Error while reading whether env is vm or not, assuming VM env : {e}")
         return is_vm
 
+    @staticmethod
+    def send_command(command: str, fail_on_error=True):
+        # This function uses subprocess() module to execute commands, scripts which are not possible
+        # to execute through any python module interfaces available in default site packages.
+        # So its usage MUST be limited and used only when no other alternative found.
+        output, error, returncode = SimpleProcess(command).run()
+        if returncode != 0:
+            print("command '%s' failed with error\n%s" % (command, error))
+            if fail_on_error:
+                sys.exit(1)
+            else:
+                return str(error)
+        return str(output)
+
+    @staticmethod
+    def call_script(script_dir: str, args: list):
+        # This function gives script execution o/p on screen, which is not fulfilled by any cortx utils function
```

Comment:
This function uses subprocess.call(), which gives script execution o/p on screen. This functionality not fulfilled by SimpleProcess() or any cortx utils function.
\#TODO merge this function code in cortx-utils